lib/control/include/clingo/control/config.hh

```diff
@@ -0,0 +1,366 @@
+#pragma once
+
+#include <clingo/util/enum.hh>
+#include <clingo/util/print.hh>
+
+#include <clasp/cli/clasp_options.h>
+
+#include <map>
+
```

Comment:
Consider including `<sstream>` , which is needed for `std::ostringstream` used in `error_()`